nucliadb/nucliadb/search/search/chat/query.py

```diff
@@ -407,10 +409,13 @@ async def maybe_audit_chat(
         audit_pb2.ChatContext(author=message.author, text=message.text)
         for message in chat_history
     ]
+    sorted_query_context = sorted_prompt_context_list(
```

Comment:
note: check if we're actually auditing the context text blocks, or just references to them